clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -5204,22 +5204,46 @@ the configuration (without a prefix: ``Auto``).
 
 .. _ReflowComments:
 
-**ReflowComments** (``Boolean``) :versionbadge:`clang-format 3.8` :ref:`¶ <ReflowComments>`
-  If ``true``, clang-format will attempt to re-flow comments. That is it
-  will touch a comment and *reflow* long comments into new lines, trying to
-  obey the ``ColumnLimit``.
+**ReflowComments** (``ReflowCommentsStyle``) :versionbadge:`clang-format 20` :ref:`¶ <ReflowComments>`
```

Comment:
have you regenerated this since you change Format.h as its still saying 20

Reply:
How embarrassing, I ran the script before I changed the version. Should be correct now, sorry!